cypress/e2e/nonParallel/data-import/importing-marc-bib-files/item-update-via-match-by-status.cy.js

```diff
@@ -177,7 +177,7 @@ describe('ui-data-import', () => {
     NewFieldMappingProfile.fillMaterialType();
     NewFieldMappingProfile.addStatisticalCode(statisticalCode, 6);
     NewFieldMappingProfile.fillPermanentLoanType('"Can circulate"');
-    NewFieldMappingProfile.fillStatus('"Available"');
+    NewFieldMappingProfile.fillStatus('Available');
```

Comment:
use ITEM_STATUSES.AVAILABLE, LOAN_TYPE_NAMES.CAN_CIRCULATE

Reply:
the same